src/Plaid/PlaidClient.cs

```diff
@@ -418,6 +418,15 @@ private async Task<TResponse> CreateResponse<TResponse>(HttpResponseMessage resp
 					try
 					{
 						plaidException = _serializer.Deserialize<PlaidException>(jsonReader);
+
+						// sometimes there is no JSON data in the response but service was still down
+						// an exception must be included
+						plaidException ??= new PlaidException()
+						{
+							ErrorMessage = "Empty response",
+							ErrorCode = ErrorCode.InternalServerError,
+							ErrorType = ErrorType.ApiError
+						};
```

Comment:
If these is no json in the response, say maybe Cloudflare gives an html 502 page. `Deserialize` will throw an exception or it will return an empty result?

Reply:
Based on logs we get a `503` from plaid but normal response logic is running as if `plaidException` is null.

Exception handler logic is like this:
```csharp
if (exception == null) {
    return null;
}
throw new Exception(); //example
```